packages/wallet/test/services/HandlesTracker.test.ts

```diff
@@ -33,15 +45,19 @@ const expectedHandleInfo = {
 describe('createHandlesTracker', () => {
   it('matches utxo$ assets to given handlePolicyIds and adds context from assetInfo$ and tip$ observables', () => {
     createTestScheduler().run(({ hot, expectObservable }) => {
-      const utxo$ = hot(     '-ab-', { a: utxo, b: utxo2 });
-      const tip$ = hot(      'a-b-', { a: ledgerTip, b: ledgerTip2 });
+      const utxo$ = hot('-ab-', { a: utxo, b: utxo2 });
```

Comment:
Did a lint rule remove these space? It's done purposely to make the marble tests readable

Reply:
there are lint added changes that I hadn't seen before, I'll fix those to to be as they were before. I haven't changed any rule.. so I am not sure why.

Reply:
b666699d85e862981740c6a00a3321c1b1bd1154 should solve this